client/app/components/Dish.js

```diff
@@ -20,21 +21,38 @@ export class Dish extends Component {
 
     componentDidMount() {
         // TODO: Check if this item is a favorite
-        isFaveDish = false; // TODO: remove this line
-
-        if (isFaveDish) {
-            this.setState({isFave: true});
-        };
+        const { data } = this.props.favoritesList;
+        const isFave = data ? data[this.props.dishID] : false;
+        this.setState({ isFave });
```

Comment:
This will set `isFave` to be `undefined` if the dish is not favorited. Is that ok?

Reply:
Ah, good point. Yes, it works but maybe I should rewrite it. Thoughts?

Reply:
You could just do 
```
const isFave = data && data[this.props.dishID] ? data[this.props.dishID] : false;
```